### PR TITLE
Updated FAQ to reflect WHP support as of April 21st 2020

### PR DESCRIPTION
**Pull request by sundhaug92**
### Proposed changes

Corrected the FAQ to reflect the fact that when using WHP it is possible to use virtualbox (and a few other hypervisors) while Docker for Desktop is installed

### CODE REVIEW

File: docker-for-windows/faqs.md [hunk 1]
```diff
@@ -113,8 +113,7 @@ Docker Desktop can run inside a Windows 10 VM running on apps like Parallels or
 
 ### Can I use VirtualBox alongside Docker Desktop?
 
-Unfortunately, VirtualBox (and other hypervisors like VMWare) cannot run when
-Hyper-V is enabled on Windows.
+Yes. VirtualBox, and a few other virtualization-solutions can run while Hyper-V is enabled on Windows as long as the optional feature [Windows Hypervisor Platform](https://docs.microsoft.com/en-us/virtualization/api/) is also enabled. Unfortunately, VMWare Workstation does not at the moment support the Windows Hypervisor Platform, but such support has been [announced](https://techcommunity.microsoft.com/t5/Virtualization/VMware-Workstation-and-Hyper-V-Working-Together/ba-p/825831) to be coming.
```
Comment (StefanScherer) [inline docker-for-windows/faqs.md:116]: I suggest to remove the last sentence as this question only covers VirtualBox.

```suggestion
Yes. VirtualBox, and a few other virtualization solutions can run while Hyper-V is enabled on Windows as long as the optional feature [Windows Hypervisor Platform](https://docs.microsoft.com/en-us/virtualization/api/) is also enabled.
```

Comment (sundhaug92) [reply docker-for-windows/faqs.md:116]: Reason for the last sentence is that the current answer also talks about VMWare

Comment (usha-mandya) [reply docker-for-windows/faqs.md:116]: @sundhaug92  We can update this entry in the FAQ to include VMware Workstation when it fully supports the Windows Hypervisor Platform. Please remove the line about VMware Workstation as it is now a forward-looking statement.

**Comment by usha-mandya** [comment]
@sundhaug92 The FAQ has been updated to reflect support for running VirtualBox along with Docker Desktop. Thanks for letting us know about this. I'm closing this PR as the changes are live now.

**Comment by sundhaug92** [comment]
Sorry for dropping the ball on adjusting this PR. Thanks for fixing the underlying issue

**Pull request closed without merging**